README.md

```diff
@@ -222,8 +222,35 @@ It returns the same type as the `list_like` argument
 ```
 
 
+#### `apply_formatters_to_sequence(formatters, <list_like>)` -> <new_list_like>
+
+This function will apply each formatter at to the list-like value, at the
+position it was supplied. It returns the same time as the `list_like` argument.
+For example:
+
+```py
+>>> from eth_utils.curried import apply_formatters_to_sequence
+
+>>> list_formatter = apply_formatters_to_sequence([bool, int, str])
+
+>>> list_formatter([1.2, 3.4, 5.6])
+[True, 3, '5.6']
+
+>>> list_formatter((1.2, 3.4, 5.6))
+(True, 3, '5.6')
+
+# it will pass through items longer than the number of formatters supplied
+>>> list_formatter((1.2, 3.4, 5.6, 7.8))
+[True, 3, '5.6', 7.8]
+```
+
+
 #### `combine_argument_formatters(*formatters)` -> lambda <list_like>: <new_list_like>
```

Comment:
Removal seems fine, but we should take it through a deprecation process unless we want another week like last week.  No rush to remove it, we just need to stop using it.

Reply:
Ok, so since we're trying to lock in the stable version, a deprecation process means removal in v2 to me. :+1: Works for me

(The only reason I thought we could "get away with" removing it right away is that this method was added in v1-beta, as opposed to the other methods that have been around since v0.)

Reply:
Lets wait until v2 for removal.  I suspect it's being used externally already even though it was only introduced based on how quickly we moved things to use the `v1-beta` releases.